vm/evm_test.go

```diff
@@ -0,0 +1,158 @@
+package vm_test
```

Comment:
`tracer` package depends on `vm` package, that's why it will show cycling dependency. So if you want the package name to be `vm` you shouldn't import `tracers` here. And tracers related functionality are tested in its package, here I would suggest do not touch the tracer related part. But if it's required, make a noop tracer copy in the file would be a approach.